test/sqlite/sqllogic_command.cpp

```diff
@@ -195,6 +195,8 @@ void LoopCommand::ExecuteInternal(ExecuteContext &context) const {
 	loop_def.loop_idx = definition.loop_start;
 	if (loop_def.is_parallel) {
 		for (auto &running_loop : context.running_loops) {
+			// FIXME: this is only checking one level down
+			// concurrent loop followed by a regular loop, with another concurrent loop inside is not prevented by this
```

Comment:
Could you show a test case where that happens? This is looping over all running loops, and trying this out seems to be working as intended:

###### test/test.test
```sql
concurrentloop i 0 100

loop j 0 100

concurrentloop k 0 100


endloop

endloop



endloop


```bash
build/debug/test/unittest test/test.test
> explicitly with message:
  Failure at :-1094795586: Nested parallel loop commands not allowed

```

(although there seems to be an issue with an uninitialized value here)

Reply:
Ah I see now, the `running_loops` contains all the levels of nesting, so yes the comment is wrong, all levels are checked 👍

Reply:
I think I misread and mistakenly thought this was checking the inner loops when we encountered a `loop` / `concurrent loop` command:

```sql
concurrent loop <loop we're checking>

loop <inner 1>

endloop

loop <inner 2>

endloop

loop endloop <inner 3>

endloop
```

But that's not what's happening, we're checking the parent loops instead